spec/defines/allow_spec.rb

```diff
@@ -7,6 +7,13 @@
         let(:title) { 'foo_bar' }
         let(:params) { {:pattern => 'localhost'} }
         it { is_expected.to contain_concat__fragment("tcpwrappers_#{title}") }
+
+        context 'with triggered nets2ddq' do 
+          let(:title) { 'foo_bar' }
+          let(:params) { { :pattern => 'localhost myhost 129.3.0.1 2001:0db8:85a3:0000:0000:8a2e:0370:7334' } }
```

Comment:
Please add an IPv4 network (e.g. 10.0.1.0/24) to the test so that we verify the transformation by simplib::nets2ddq (e.g. 10.0.1.0/255.255.255.0) is properly processed by simplib::bracketize.

Reply:
roger